toolkit-core.sln

```diff
@@ -1,7 +1,6 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.705
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29911.84
```

Comment:
If I open this in VS 2017, won't this version number just change right back?

Reply:
I'm not sure how it will react to a higher version number. This happens automatically when you save the solution file.

Reply:
It doesn't seem to change it and I need to update anyway.